test/e2e/node/apparmor.go

```diff
@@ -39,11 +39,11 @@ var _ = SIGDescribe("AppArmor", func() {
 		})
 
 		ginkgo.It("should enforce an AppArmor profile", func() {
-			common.CreateAppArmorTestPod(f, false, true)
+			e2esecurity.CreateAppArmorTestPod(f, false, true)
```

Comment:
Could you please refactor the definiton of `CreateAppArmorTestPod` to take a Client Interface instead of taking the framework as an argument?
In order to prevent circular dependencies we need to make sure no functions need the test framework.

Reply:
done, thanks